app/services/data_migrations/remove_application_choice_in_the_incorrect_cycle.rb

```diff
@@ -1,16 +1,15 @@
 module DataMigrations
-  class RemovePreviousCyclesCoursesFromApplicationsInTheCurrentCycle
-    TIMESTAMP = 20210528131818
+  class RemoveApplicationChoiceInTheIncorrectCycle
+    TIMESTAMP = 20210602144637
     MANUAL_RUN = false
 
     def change
       ApplicationChoice
       .joins(:application_form, course_option: [:course])
```

Comment:
Should we consider `current_course_option` too?

Reply:
So my vote is no here is no, just because there's two and i think we should investigate how that happened. It seems like you it shouldn't be possible.

I 100% think the invariant check should be updated to check though!